config/logging.php

```diff
@@ -89,6 +89,15 @@
             'driver' => 'errorlog',
             'level' => 'debug',
         ],
+
+        'null' => [
+            'driver' => 'monolog',
+            'handler' => NullHandler::class,
+        ],
```

Comment:
omg what is `null` logging?

Reply:
They go nowhere! Haha <https://github.com/laravel/laravel/pull/5106>